requirements.txt

```diff
@@ -9,14 +9,15 @@ cookiecutter==2.4.0
 Django==4.2.10
 django-browser-reload==1.12.0
 django-tailwind==3.6.0
+python-dotenv
 idna==3.4
 Jinja2==3.1.3
 joblib==1.3.2
 markdown-it-py==3.0.0
 MarkupSafe==2.1.3
 mdurl==0.1.2
 nltk==3.8.1
-psycopg2
+psycopg2-binary
```

Comment:
why "-binary"?

Reply:
I got errors every time i tried to download pyscopg2 which said get the binary version

Reply:
let's include both then